test/unit/test_ksw_global2_wave.cpp

```
@@ -0,0 +1,221 @@
+void run_parity(const std::vector<TestPair> &pairs, const int8_t *mat) {
+    int score_mism = 0, ncigar_mism = 0, cigar_mism = 0;
+    long compared = 0;
+    for (const auto &p : pairs) {
+        const int floor = band_floor(p);
+        for (const GapSet &g : kGapSets) {
+            for (int wt : kBandTargets) {
+                const int w = wt > floor ? wt : floor;
+                const G2Result got = run_g2(ksw_global2, p, mat, w,
+                                            g.o_del, g.e_del, g.o_ins, g.e_ins);
+                const G2Result ref = run_g2(ksw_global2_scalar_ref, p, mat, w,
+                                            g.o_del, g.e_del, g.o_ins, g.e_ins);
```

Cppcheck (2.21.0)

[style] The function '_mm_movemask_epi16' is never used.

(unusedFunction)